src/main/java/edu/wpi/grip/core/operations/composite/ThresholdOperation.java

```diff
@@ -0,0 +1,33 @@
+package edu.wpi.grip.core.operations.composite;
+
+import edu.wpi.grip.core.Operation;
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_core.Scalar;
+
+import java.util.Optional;
+
+public abstract class ThresholdOperation implements Operation {
```

Comment:
This should probably also implement `getIcon()` so all of the threshold operations have icons and not just RGB